### ISSUE TITLE
NSBundle loadNibNamed doesn't consider language-specific subdirs

### ISSUE THREAD

**Issue by LubosD** [darlinghq/darling#724]
[pearCalc](http://www.pearworks.com/pages/software.html) contains `MainMenu.nib` only under the `English.lproj` subdir.

Cocotron's NSBundle has `lookInDirectories` that solves this, but our NSBundle doesn't do that and fails.

**Comment by LubosD** [comment]
Working around this doesn't make it work either, because Cocotron's NSNib.m looks for `keyedobjects.nib` and this app has only `objects.nib`.

**Comment by bugaevc** [comment]
`pathForResource:ofType:` should handle language subdirectories automatically, doesn't it?

**Comment by LubosD** [comment]
You're right, I was confused - I'll change the issue. But it doesn't load `objects.nib`...

**Comment by LubosD** [comment]
Classes that don't support unkeyed coding:

- [ ] NSActionCell
- [ ] NSBox
- [ ] NSBrowser
- [ ] NSButtonCell
- [ ] NSButton
- [ ] NSCell
- [ ] NSClipView
- [ ] NSColorWell
- [ ] NSComboBoxCell
- [ ] NSControl
- [ ] NSDatePickerCell
- [ ] NSFont
- [ ] NSFormCell
- [ ] NSImage
- [ ] NSImageCell
- [ ] NSMatrix
- [ ] NSOutlineView
- [ ] NSParagraphStyle
- [ ] NSPathCell
- [ ] NSPopUpButtonCell
- [ ] NSProgressIndicator
- [ ] NSScroller
- [ ] NSScrollView
- [ ] NSSliderCell
- [ ] NSSplitView
- [ ] NSTableColumn
- [ ] NSTableView
- [ ] NSTabView
- [ ] NSTabViewItem
- [ ] NSText
- [ ] NSTextFieldCell
- [ ] NSViewController
- [ ] _NSManagedProxy
- [ ] NSButtonImageSource
- [ ] NSClassSwapper
- [ ] NSCustomObject
- [ ] NSCustomResource
- [ ] NSCustomView
- [x] NSIBObjectData
- [ ] NSIBUserDefinedRuntimeAttributesConnector
- [ ] NSNibBindingConnector
- [ ] NSNibConnector
- [ ] NSNibHelpConnector
- [ ] NSWindowTemplate
- [ ] NSCollectionView
- [ ] NSColor
- [ ] NSArrayController
- [ ] NSDictionaryController
- [ ] NSObjectController
- [ ] NSUserDefaultsController
- [ ] NSDrawer
- [x] NSMenu
- [ ] NSMenuItem
- [ ] NSOpenGLPixelFormat
- [ ] NSOpenGLView
- [ ] NSSegmentedCell
- [ ] NSSegmentItem
- [ ] NSLayoutManager
- [ ] NSTextContainer
- [ ] NSTextStorage_concrete
- [ ] NSTextTab
- [ ] NSTextView
- [ ] NSTextViewSharedData
- [ ] NSToolbar
- [ ] NSToolbarItem

**Comment by LubosD** [comment]
Correct decoder impl for Apple's unkeyed format: https://github.com/depth42/MEUnarchiver/blob/master/MEUnarchiver.m
